content/organization/events/speakers/index.en.md

```diff
@@ -27,6 +27,47 @@ weight: 53
 
 * Remember the [rules about Prioritization of Underrepresented/Minority Genders](/about/mission/#r-ladies-rules--guidelines).
 
+* In line with R-Ladies’ Mission Statement and with our [Black Live Matter Statement](https://blog.rladies.org/post/2020-06-06-blm/) 
+it is our obligation to make efforts to ensure the participation and representation in our events of generally excluded groups, taking into account the intersectionality of gender, race, language and geography, for this reazons we request:
+
+  - Not host all the events with white speakers in the same chapter during the year.  
+  - Avoid all white panels.
+
+### How to find speakers? 
+- You can use the [R-Ladies Directory](https://rladies.org/directory/) to search and find non white, native anglophone, high income level countries speakers.
+- You can reach other communities to request for speakers information (see lists below).
+- You can reach other R-Ladies in the R-Ladies Community Slack.
+- You can use others directories to look for speakers (see list below). 
+
```

Comment:
- For online events, we encourage local chapters to reach out beyond their local community to invite R-Ladies from different parts of the world as speakers.